src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1117,6 +1117,7 @@ protected override void GetItem(string path)
 
             try
             {
+#if !UNIX
                 bool retrieveStreams = false;
                 FileSystemProviderGetItemDynamicParameters dynamicParameters = null;
```

Comment:
Which is more likely - somebody copying a script from Windows to Linux that uses `-Stream`, or somebody on Linux using parameter completion, sees `-Stream` as a possible option, and wastes time trying to learn what it does?

I'm thinking it's better overall to not suggest parameters on platform if they aren't actually supported.